### PR TITLE
Add unit tests for CassandraTableSensor, CassandraRecordSensor and WebHdfsSensor

### PR DESCRIPTION
**Pull request by rootcss**
Add unit tests for CassandraTableSensor, CassandraRecordSensor and WebHdfsSensor. Mentioned as missing unit tests in issue #8278

---
Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Target Github ISSUE in description if exists
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

File: tests/providers/apache/cassandra/sensors/test_record.py [hunk 1]
```diff
@@ -21,15 +21,38 @@
...
+            task_id='test_task',
+            cassandra_conn_id=TEST_CASSANDRA_CONN_ID,
+            table=TEST_CASSANDRA_TABLE,
+            keys=None,
         )
-        sensor.poke(None)
-        mock_hook.return_value.record_exists.assert_called_once_with('t', {'foo': 'bar'})
+        exists = sensor.poke(dict())
+
+        self.assertTrue(exists)
```
Comment (mik-laj) [inline tests/providers/apache/cassandra/sensors/test_record.py:55]: Can you add a test that will check False?

File: tests/providers/apache/cassandra/sensors/test_table.py [hunk 2]
```diff
@@ -21,14 +21,36 @@
...
+        sensor = CassandraTableSensor(
+            task_id='test_task',
+            cassandra_conn_id=TEST_CASSANDRA_CONN_ID,
+            table=TEST_CASSANDRA_TABLE_WITH_KEYSPACE,
         )
-        sensor.poke(None)
-        mock_hook.return_value.table_exists.assert_called_once_with('t')
+        exists = sensor.poke(dict())
+
+        self.assertTrue(exists)
```
Comment (mik-laj) [inline tests/providers/apache/cassandra/sensors/test_table.py:69]: Can you add a test that will check False?

File: tests/providers/apache/hdfs/sensors/test_web_hdfs.py [hunk 3]
```diff
@@ -15,27 +15,28 @@
...
+    @mock.patch('airflow.providers.apache.hdfs.hooks.webhdfs.WebHDFSHook')
+    def test_poke(self, mock_hook):
+        sensor = WebHdfsSensor(
+            task_id='test_task',
+            webhdfs_conn_id=TEST_HDFS_CONN,
+            filepath=TEST_HDFS_PATH,
+        )
+        exists = sensor.poke(dict())
+
+        self.assertTrue(exists)
```
Comment (mik-laj) [inline tests/providers/apache/hdfs/sensors/test_web_hdfs.py:39]: Can you add a test that will check False?

**Comment by rootcss** [comment]
@mik-laj I've added tests where hooks return negative values.

**Review by mik-laj — APPROVED** [review]

**Comment by mik-laj** [comment]
Thanks. What's next? 🐈

**Comment by rootcss** [comment]
> Thanks. What's next?

Thanks @mik-laj. If you have anything in mind, please let me know. Otherwise, I'll go through the issues anyway.

**Comment by mik-laj** [comment]
We're close to finishing this ticket. We only have to do 10 tests. Some files are WIP.

**Comment by rootcss** [comment]
> We're close to finishing this ticket. We only have to do 10 tests. Some files are WIP.

Sure, I'll try to finish the ones that are not picked from https://github.com/apache/airflow/issues/8278 yet.

**Pull request merged**